src/components/BottomSheet.tsx

```diff
@@ -58,6 +55,13 @@ export const BottomSheet = ({children, collapsedContent, extraContent}: BottomSh
   const onOpenEnd = useCallback(() => setIsExpanded(true), []);
   const onCloseEnd = useCallback(() => setIsExpanded(false), []);
 
+  const {width, height} = useWindowDimensions();
+  const snapPoints = [height, Math.max(width, height) * (extraContent ? 0.3 : 0.2)];
+
+  useEffect(() => {
+    bottomSheetRef.current?.snapTo(isExpanded ? 0 : 1);
+  }, [width, isExpanded]);
```

Comment:
This isn't ideal - unfortunately without it when the bottom sheet is collapsed, on rotation it sometimes would disappear.